ena-submission/scripts/ena_submission_helper.py

```diff
@@ -129,6 +129,18 @@ def get_project_xml(project_set):
         "PROJECT": dataclass_to_xml(project_set, root_name="PROJECT_SET"),
     }
 
+def reformat_authors(authors: str) -> str:
+    """This function reformats the Loculus authors string to the format expected by ENA"""
```

Comment:
```suggestion
    """This function reformats the Loculus authors string to the format expected by ENA
    Loculus format: `John A. Doe; Jane B. C. Roe`
    EMBL expected: `Doe J.A., Roe J.B.C.`
    
    EMBL spec: "The names are listed surname first followed by a blank
      followed by initial(s) with stops.
      Occasionally the initials may not be known,
      in which case the surname alone will be listed.
      The author names are separated by commas
      and terminated by a semicolon; they are not split between lines."    
    See section "3.4.10.6: The RA Line" here: https://raw.githubusercontent.com/enasequence/read_docs/c4bd306c82710844128cdf43003a0167837dc442/submit/fileprep/flatfile_user_manual.txt"""
```

Adding some extra docs here to simplify checking this function and refactoring in future, also writing tests etc.

Reply:
Unclear about the Loculus input format used here.

Thinking about it, since we need to separate first names and last names, and last names can sometimes be multiple names that are whitespace separated, our format isn't unambiguous. Maybe we want to think about how we keep names unambiguous internally, something like:
`LASTNAME [OTHERLASTNAME...], FIRSTNAME [MOREFIRSTNAMES...]; ...` could work. Not a blocker for this PR, we can already improve based on our current system, just assuming single last names for now which should be correct in most cases (except for Spanish names)

Reply:
It seems that the above spec is exactly the input format you wrote the function for - but I'm not sure we're using that one at this point